### PR TITLE
Add TUO for Contact Sensor Driver

### PR DESCRIPTION
**Pull request by sam-gabbay**

### CODE REVIEW

**Comment by lelandblue** [comment]
Hello @sam-gabbay. Thank you for your pull request. Please sign the CLA. 

Have a good day.

**Comment by sam-gabbay** [comment]
@lelandblue it has been signed, I am not sure why it is showing as _not signed yet_.

File: drivers/SmartThings/matter-sensor/fingerprints.yml [hunk 1]
```diff
@@ -16,6 +16,12 @@ matterManufacturer:
     vendorId: 0x130a
     productId: 0x004D
     deviceProfileName: contact-battery
+  #TUO
+  - id: "TUO/ContactDoorAndWindow"
+    deviceLabel: TUO Contact Sensor
+    vendorId: 0x141E
+    productId: 0x02
```
Comment (ctowns) [inline drivers/SmartThings/matter-sensor/fingerprints.yml:23]: Could you add the leading zeros to the `productId` so it has 4 digits like the other examples? e.g.
`productId: 0x0002`

**Review by ctowns — COMMENTED** [review]
One small change, otherwise this looks good

**Comment by sam-gabbay** [comment]
Done @ctowns

**Review by ctowns — APPROVED** [review]

**Comment by sam-gabbay** [comment]
@ctowns @lelandblue can you advise what is needed at this point?

**Comment by ctowns** [comment]
@sam-gabbay Can you please sync your forked repo with the latest from the main repo so that the units tests will pass?

**Comment by sam-gabbay** [comment]
@ctowns Done!

**Comment by ctowns** [comment]
> @ctowns Done!

Thanks @sam-gabbay! All tests are passing now, this looks good to me.

**Comment by lelandblue** [comment]
Hello @TUOAccessoriesCI Can you please sign the CLA please? We will need that signed before we can merge the code. 

Thank you.

**Pull request merged**